datasetinsights/estimators/__init__.py

```diff
@@ -1,6 +1,12 @@
 from .base import Estimator
 from .deeplab import DeeplabV3
 from .densedepth import DenseDepth
-from .faster_rcnn import FasterRCNN
+from .faster_rcnn import FasterRCNN, convert_bboxes2canonical
 
-__all__ = [Estimator, DeeplabV3, DenseDepth, FasterRCNN]
+__all__ = [
+    Estimator,
+    DeeplabV3,
+    DenseDepth,
+    FasterRCNN,
+    convert_bboxes2canonical,
+]
```

Comment:
**all** should be a list of strings instead of a list of class names.
https://docs.python.org/3/tutorial/modules.html#importing-from-a-package